docs/dev-guide/integration/rest-api/consumer/applicability.rst

```diff
@@ -59,11 +59,21 @@ to the given repositories, applicability data is generated for them as well.
 Generated applicability data can be queried using 
 the `Query Content Applicability` API described below.
 
-The API will return a :ref:`group_call_report`. Users can check whether the applicability
-generation is completed using group id in the :ref:`group_call_report`. The `_href` in the
-:ref:`group_call_report` will point to the root of `task_group` resource. However, this API
-endpoint currently returns 404 in all cases. You can append '/state-summary/' to the URL and
-perform a GET request to retrieve a :ref:`task_group_summary`.
+There are two ways to regenerate applicability for consumers bound to a repo. The computation can
+be done linearly or in parallel. An optional `parallel` argument determines which method is used.
```

Comment:
Consider replacing `An optional`... with: `The default is linear, but an optional `parallel` argument can enable parallel calculation.`